### DISCUSSION TITLE
More information needed on .txt file in the folder

### DISCUSSION THREAD

**Discussion by thomprl** [CMSgov/hospital-price-transparency#5] [Q&A] [answered]
We need additional information on the .txt file that will be placed in the root folder on the website.

1. What does this .txt file need to be named?
2. Define what is meant by root folder.  Does this .txt file need to go in the root folder of the public facing website or the price transparency hosting site? For example myhospital.com/??.txt or does it need to go on the site that hosts our price transparency? An outside company hosts our Price Transparency. For example: mypricetransparency.com/??.txt
3. The link in the footer that is labeled Price Transparency what do we do if we have multiple facilities? Does this need to go to another page that allows you to select which facility you are interested in?

**Comment by shaselton-usds** [comment, marked as answer]
@thomprl  Thank you for your questions. Please see the responses below each question.

##### What does this .txt file need to be named?

Name the file “cms-hpt.txt”. Required information and technical specifications for the .txt file, as well as an example .txt file, are available on the HPT Tools Webpage here: https://cmsgov.github.io/hpt-tool/txt-generator.

##### Define what is meant by root folder. Does this .txt file need to go in the root folder of the public facing website or the price transparency hosting site? For example [myhospital.com/??.txt](http://myhospital.com/??.txt) or does it need to go on the site that hosts our price transparency? An outside company hosts our Price Transparency. For example: [mypricetransparency.com/??.txt](http://mypricetransparency.com/??.txt)

Place the TXT file on the root of the domain of the public website your hospital has selected to host its machine-readable file (MRF), without regard to page structure. As an example, a hospital with the website https://hospital.com/ would locate its file at https://hospital.com/cms-hpt.txt. Additional details about the root folder can be found on the HPT Tools Webpage here: https://cmsgov.github.io/hpt-tool/txt-generator.


##### The link in the footer that is labeled Price Transparency what do we do if we have multiple facilities? Does this need to go to another page that allows you to select which facility you are interested in?

In the CY 2024 OPPS/ASC Final Rule, CMS indicated having a footer link to a page with multiple links to the various MRFs would help consolidate and aggregate the information, provide consumers with a standard pathway to find the information, provide some uniformity, and ensure that the MRFs and their data contents are easily accessible. Refer to the discussion at [88 FR 82113](https://www.federalregister.gov/d/2023-24293/p-5086).

**Reply by jzh512** [reply]
On the HPT Tools Webpage, the Hospital Location Name in the generator is the first required line.  For health systems with multiple facilities, will it be sufficient to put in the health system's name in the generator rather than generate a .txt file for the individual facilities?  Thank you.

**Reply by shaselton-usds** [reply]
@jzh512 Thank you for your question.

If the MRF the txt file links to contains standard charge information for more than one location, create an entry for each of the inpatient locations and standalone emergency hospitals in the TXT file. Please refer to the required information and technical specifications for the txt file available on the HPT Tools Webpage here: https://cmsgov.github.io/hpt-tool/txt-generator.

**Reply by jzh512** [reply]
Thank you for your prompt reply. 

1. Confirming that within the single txt file, that the detail for each location can be included, and that an individual txt file is not required of each location (e.g., [www.hospital.org/cms-hpt.txt] vs. [www.hospital.org/location-HospitalA/cms-hpt.txt])

2. Additionally, for the POC in the HPT Tools Webpage txt generator, is it acceptable to list the POC as a team rather than an individual? Is the intent of the POC to provide authority from the hospital/health system to answer questions related to MRF compliance?

**Reply by UCIANGIE** [reply]
I have the same question, is CMS allowing for a group, or department name, along with a group email address to be submitted under the POC field?  
Many organizations are wanting to approach this in this manner to ensure that communication is not lost or limited to one person.  There have been instances where time sensitive information was directed to an individual and that individual is no longer at the organization or may have changed roles and the communication is lost along the way.  The price transparency information is expected to be maintained once a year, so there could be a gap if this is limited to a single person.

**Comment by RidenourK** [comment]
@jzh512 @UCIANGIE Thank you for your questions. 

Your hospital should create a single TXT file and include within it the links for each of your hospital location MRFs.  [The schema](https://cmsgov.github.io/hpt-tool/txt-generator/) describes in detail the required file structure and details on each field, including how to encode separate entries.  Alternatively, the TXT file generator can guide you on how to include multiple entries in a single TXT file by clicking ‘add’ after each entry.

You must include the name and email address of the point of contact(s) you have designated to answer technical questions about your hospital’s MRF, and the data encoded in it (for example:  John Doe or ABC Hospital MRF Team and jdoe@ABChospital.com or MRFteam@ABChospital.com).

Please refer to the updated guidance on the TXT file and example TXT files on the TXT Generator page of the [Hospital Price Transparency Tools site](https://cmsgov.github.io/hpt-tool/) for information on how to include multiple locations and indicate POC information in the TXT file.
